book_source/03_topical_pages/94_docker/02_quickstart.Rmd

```diff
@@ -1,4 +1,37 @@
-## The PEcAn docker install process in detail {#docker-quickstart}
+## Quick-start docker install {#docker-quickstart}
+
+```bash
+git clone git@github.com/pecanproject/pecan
+cd pecan
+
+# start database
+docker-compose -p pecan up -d postgres
+
+# add example data (first time only)
+docker-compose run --rm bety initialize
```

Comment:
this has changed in the bety documentation and should be reflected here as well.

Reply:
we now leverage of the pecan/db container to speed up the process of loading data. No need to initialize using bety container.

```
docker run --rm --network pecan_pecan pecan/db
```

Reply:
@robkooper this be a docker-compose command?

Reply:
no, think of docker-compose as services taht should be running continously. This command is a one time initialization of the data. Once you have done it once, you can use `docker-compose run --rm bety sync` to synchronize the data.

Reply:
On a related note, why doesn't the 'Docker Quickstart' use `docker.sh`?

Reply:
docker.sh is the script to build the docker containers.

Reply:
I didn't see that. Thank you for clarifying.

Reply:
That should be somewhere, however I think the quickstart really should be a "use existing containers" start. Not the and now lets build the containers before you get started start.

So I like how this is done right now by @dlebauer